brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1016 Update checkbox and radio default size to 10px for cf7 designer block

### Description
Update checkbox and radio default size to 10px for cf7 designer block

### Types of changes
Improvement ( Non-breaking change )

### How has this been tested?
- Add new Contact Form 7 Designer block 
- Make sure the form has a checkbox and radio buttons 
- On default settings, the size for the checkbox and radio should be 10px. 

### Checklist:
